lighthouse-core/test/audits/byte-efficiency/unminified-javascript-test.js

```diff
@@ -58,30 +67,67 @@ describe('Page uses optimized responses', () => {
     }));
 
     expect(results).toMatchObject([
-      {url: 'foo.js', wastedPercent: 57, wastedKB: 11},
-      {url: 'other.js', wastedPercent: 53, wastedKB: 27},
+      {url: 'foo.js', wastedPercent: 56, wastedKB: 11},
```

Comment:
why did this change?

Reply:
whitespace changes in the provided network record contents. the `content: ` eats up some of the ws :)